netskrafl.py

```diff
@@ -2258,8 +2279,10 @@ def oauth2callback():
         # 500 - Internal server error
         return jsonify({"status": "invalid", "msg": "Missing CLIENT_ID"}), 500
 
-    token = request.form.get("idToken", "")
-    csrf_token = request.form.get("csrfToken", "")
+#  or request.json['idToken']
+# or request.json['csrfToken']
+    token = request.form.get("idToken", "") or request.json['idToken']
```

Comment:
Should probably be `request.json.get("idToken")`, otherwise an exception will be raised at run-time if token is missing